### DISCUSSION TITLE
Move "center" downwards?

### DISCUSSION THREAD

**Discussion by moving-bits** [mapsforge/vtm#1098] [Q&A] [answered]
"Center" is currently at 50% height / 50% width position. Is there a way to move the "center" downwards, eg. to 10% from the bottom, without having to calculate the position difference manually in the app?

(Ideally nothing else needs to be adapted in the app, and map rotation should still work, but be relative to the new "center" position.)

Use case is sort of a "driving mode", where map auto rotation is active, current position is app. 10% from screen's bottom, so that you can see as much as possible from the map that lays ahead. See https://github.com/cgeo/cgeo/issues/15257.

**Comment by devemux86** [comment, marked as answer]
This can be adjusted with `map().viewport().setMapViewCenterY(float)`.

_"Set pivot vertical relative to view center in [-1, 1].
e.g. pivotY 0.5 is usually preferred for navigation, moving center to 25% of view height."_

**Reply by moving-bits** [reply]
This is great news, thanks for the quick pointer, I'll experiment with it.
